automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -47,7 +53,7 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
 
     def linesWithoutIPs(file: File) = {
       import scala.collection.JavaConverters._
-      Files.readAllLines(left.toPath).asScala map { _.replaceAll("(\\d+.){3}\\d+", "<IP>") }
+      Files.readAllLines(file.toPath).asScala map { _.replaceAll("(\\d+.){3}\\d+", "<IP>") }
```

Comment:
So... `compareFilesExcludingIPs` all this time hasn't been doing anything because it was comparing the left file with itself. 🤦‍♂️

Reply:
doh!